tests/failpoints/cases/test_merge.rs

```diff
@@ -467,4 +467,5 @@ fn test_node_request_snapshot_reject_merge() {
     let target = cluster.get_region(target_region.get_start_key());
     assert_eq!(target, target_region);
     fail::remove(region_gen_snap_fp);
+    drop(cluster);
```

Comment:
But cluster is dropped automatically at the end of the test.

Reply:
Yes, but seems it sends a message to `rx` after `rx` is dropped, then causes a `SendError`. So I think ensuring `cluster` is dropped earlier than `rx` would solve this problem.

Reply:
LGTM, could you add some comments?

Reply:
Sure.